Utils/Library.cs

```diff
@@ -0,0 +1,273 @@
+                Action = "Book Removed",
+                BookTitle = book.Title
+            };
+            logManager.AddLogEntry(logEntry);
+        }
+
+        /// <summary>
+        /// Checks if a user is eligible to issue a book based on their type and the number of books they have borrowed.
+        /// </summary>
+        /// <param name="user">The user to check eligibility for.</param>
+        /// <returns>True if the user is eligible; otherwise, false.</returns>
+        private bool IsEligibleToIssue(User user)
+        {
+            if (user.Type == UserType.Student && user.BorrowedBooks.Count >= 3)
+            {
+                Console.WriteLine("Students can issue 3 books at a time.");
+                return false;
+            }
+
+            if (user.Type == UserType.Teacher && user.BorrowedBooks.Count >= 10)
+            {
+                Console.WriteLine("Teachers can issue 10 books at a time.");
+                return false;
+            }
+            return true;
+        }
+
+        /// <summary>
+        /// Issues a book to a user.
+        /// </summary>
+        /// <param name="book">The book to issue.</param>
+        /// <param name="user">The user to whom the book is issued.</param>
+        /// <param name="admin">The admin issuing the book.</param>
+        public void IssueBook(Book book,User user,Admin admin)
+        {
+            if(!IsEligibleToIssue(user))
+            {
+                return;
+            }
+
+            if (!book.IsAvailable)
+            {
+                Console.WriteLine("The book is not available for borrowing.");
+                return;
+            }
+
+            book.IsAvailable = false;
+            book.Issuees.Add(user);
+            book.SetIssueDate(DateTime.Now); 
+            user.BorrowedBooks.Add(book);
+            book.Issuer = admin.AdminName;
+            Console.WriteLine($"Book '{book.Title}' issued to {user.Name} successfully.\nIssuer: {book.Issuer}.");
+
+            LogEntry logEntry = new LogEntry
+            {
+                Timestamp = DateTime.Now,
+                Action = "Book Issued",
+                UserName = user.Name,
+                BookTitle = book.Title
+            };
+            logManager.AddLogEntry(logEntry);
+        }
+
+        /// <summary>
+        /// Returns a book borrowed by a user.
+        /// </summary>
+        /// <param name="user">The user returning the book.</param>
+        /// <param name="book">The book being returned.</param>
+        public void ReturnBook(User user,Book book) 
+        {
+            double fine = 0;
+
+            if (!book.Issuees.Contains(user))
+            {
+                Console.WriteLine("The book was not issued by this user.");
+                return ;
+            }
+            book.IsAvailable = true;
+            user.BorrowedBooks.Remove(book);
+            TimeSpan duration = DateTime.Now - book.IssueDate.Value; 
+            if (duration.Days > 7)
```

Comment:
You should create separate constants file for storing all these constants. To make you code even more readable you can create separate constants for all the console logs. For eg. "MenuOptions", "ResponseMessages" etc.